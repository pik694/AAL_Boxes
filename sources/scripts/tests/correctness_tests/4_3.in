4
71 18 35
45 2 77
73 27 42
69 21 5

