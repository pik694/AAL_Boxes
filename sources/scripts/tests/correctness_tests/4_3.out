1 3 0 2 
141687->89712
